app/parse.py

```
@@ -15,8 +29,106 @@ class Product:
     num_of_reviews: int
 
 
+PRODUCT_NAME = [field.name for field in fields(Product)]
+
+
+def get_driver() -> webdriver:
+    options = webdriver.ChromeOptions()
+    # options.add_argument("--headless")
+
+    driver = webdriver.Chrome(options=options)
+    return driver
+
+
+def cookies(driver: webdriver) -> None:
+    try:
+        wait = WebDriverWait(driver, 15)
+        accept_cookie = wait.until(
+            EC.element_to_be_clickable((
+                By.CLASS_NAME, "acceptCookies"
+            ))
+        )
+
+        accept_cookie.click()
+        time.sleep(5)
+    except TimeoutException:
+        print("cookie doesn't found")
+
+
+def scroll_more(driver: webdriver) -> None:
+    while True:
+        try:
+            wait = WebDriverWait(driver, timeout=15)
+            more = wait.until(
+                EC.element_to_be_clickable(
+                    (
+                        By.CLASS_NAME, "ecomerce-items-scroll-more")
+                )
+            )
+            more.click()
+
+        except Exception:
+            break
```

`scroll_more` catches a broad `Exception` which can hide real errors (e.g., parsing errors). Narrow the exception handling to the expected ones (e.g., `TimeoutException` or `NoSuchElementException`) and let unexpected exceptions propagate so failures are visible in CI.